go.mod

```diff
@@ -4,7 +4,8 @@ go 1.13
 
 require (
 	github.com/blang/semver v3.5.1+incompatible
+	github.com/go-git/go-git/v5 v5.3.0
 	github.com/otiai10/copy v1.0.1
 	github.com/otiai10/curr v0.0.0-20150429015615-9b4961190c95 // indirect
-	github.com/go-git/go-git/v5 v5.1.0
+	gopkg.in/src-d/go-git.v4 v4.13.1
```

Comment:
There seems to be a version conflict here. Above we are setting the version to 5.3.0 and on line 10 we are setting the old unsupported version of [v4, 4.13.1](https://github.com/src-d/go-git).